src/test/java/io/strimzi/kafka/metrics/integration/TestStreamsMetricsIT.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.metrics.integration;
+
+import io.strimzi.kafka.metrics.KafkaPrometheusMetricsReporter;
+import io.strimzi.kafka.metrics.TestUtils;
+import io.strimzi.test.container.StrimziKafkaContainer;
+import org.apache.kafka.clients.admin.Admin;
+import org.apache.kafka.clients.admin.AdminClientConfig;
+import org.apache.kafka.clients.admin.NewTopic;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.common.serialization.StringSerializer;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.containers.GenericContainer;
+
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class TestStreamsMetricsIT {
+
+    private StrimziKafkaContainer broker;
+    private Map<String, String> env;
+
+    @BeforeEach
+    public void setUp() throws Exception {
+        broker = new StrimziKafkaContainer()
+                .withKraft()
+                .withNetworkAliases("kafka");
```

Comment:
Extract this network `kafka` alias into a TestUtils so it can be reused across all integration tests.